Project-Descriptions-and-Plans/LLVM-Toolchain/llvm-toolchain-pl.md

```diff
@@ -0,0 +1,314 @@
+  - Typically lacks detail such as a feature list, architecture or complete project team.
+  - If PC approved by the TWG, the project team completes the project description in the **PL proposal**.
+  - The PC as approved is retained in the programs repository*
+
+- The **PL**:
+  - Is a separate document to the PC, which it updates and completes.
+  - The architecture, major features and project phases (the "what" - but not necessarily final level of detail)
+  - The project team is identified
+  - If PL approved by the TWG, the project team completes the feature specification and project plan in the **Plan Approved (PA) proposal**. 
+
+The flow for **OpenHW software projects** whose scope/target is a family of related targets (such as CORE-V 32 bit 4 stage cores):
+- PC and PL are written as top-level projects addressing the target family.
+- Any PC and PL sections intended for a particular target, e.g., preliminary feature list, are identified as target-specific
+- The PA describes a "subproject" for a particular target. Each PA should contain the full feature list and project plan for a particular target
+- There would normally be multiple PA sub-projects for top-level SW projects
+
+The flow for **OpenHW Advanced RISC-V Verification Methodology (ARVM)** subprojects:
+- The ARVM PC is written as a top-level project addressing multiple ARVM subprojects
+- The PL and PA describe an ARVM "subproject". The PA  contains the full feature list and project plan for the subproject
+- There will be multiple PL/PA subprojects for the ARVM top-level project
+
+**Part 2, PL fields:**
+The PL proposal explains the "what". Some of it can be updated directly from the PC proposal -->
+
+# Title of Project - "Clang/LLVM tool chain for CORE-V"
+# Project Launch Proposal
+## Date of proposal - 2022-11-28
+## Author(s) - Jeremy Bennett, Charlie Keaney
+
+## Summary of project
+
+This proposal is for a Clang/LLVM compiler tool chain for CORE-V.  Under the new program structure, this is the overall framework for all Clang/LLVM tool chain development for CORE-V.  Separate "Plan Approved" proposals will address individual target development.  As such there is minimal work directly associated with this Project Launch proposal.
+
+Support for the Software TG's primary goal of developing a thriving commercial ecosystem is a matter for the detailed planning of individual "Plan Approved proposals".
+
+In order to support the Software TG secondary goal of upstreaming all open source tool developments:
+
+- all Clang/LLVM development will be kept compliant with the LLVM Foundation coding and quality assurance standards;
+- the implementation will follow the upstream tool design and coding conventions; and
+- any vendor-specific modifications or additions will be duly isolated into extensions, so as to maintain a fully functional common open source code base.
```

Comment:
This OpenHW project is not adding vendor-specific modifications so I'm not sure how to interpret this comment

Reply:
For the purposes of GCC, OpenHW is a vendor.